xinference/core/restful_api.py

```diff
@@ -266,11 +274,51 @@ def serve(self):
             methods=["POST"],
             response_model=ChatCompletion,
         )
-        app.include_router(self._router)
-        app = gr.mount_gradio_app(app, self._gradio_block, path="/")
+
+        self._router.add_api_route(
+            "/v1/gradio/{model_uid}", self.build_interface, methods=["POST"]
```

Comment:
You might prefer to design the API in a way that's unrelated to the library we rely on.

How about `/v1/ui/model_uid`?